src/ignitus-StudentSignUp/interest/Components/interest.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import progress from '../../../ignitus-Assets/Images/progressPage2.png';
+import interestsImage from '../../../ignitus-Assets/Images/interestsImage.png';
+import {withErrorBoundary} from '../../../ignitus-Internals';
+
+import '../Styles/style.scss';
+
+const interest = () => {
```

Comment:
You can write it in such a way,  ```
const interest = () => (
   
<div className="interest">

   ......
</div>


)
```

you can omit return along with `parenthesis`.

Reply:
@divyanshu-rawat some of the hound issues are related to class names that they should be lowercase with hyphen between them. I can leave them right?

Reply:
You can ignore hound suggestions related to lowercase naming.

Reply:
@divyanshu-rawat what about those related to color literals and labels?

Reply:
that's fine you can leave them nevertheless we will migrate to `ts` and `emotion` in future so `color literals` will be replaced by constants itself. :)